bot_cli/__main__.py

```diff
@@ -41,6 +44,16 @@ def main(address_book):
                 commands.add_contact(args)
             case "add-note":
                 commands.add_note(notebook, args)
+            case "update-note":
```

Comment:
We have `change-note` command in documentation for it to be consistent with the `change-phone` command that we already have

Reply:
update delete
replace rename to change note